addons/mail/static/src/js/extended_chat_window.js

```diff
@@ -39,6 +39,11 @@ return ChatWindow.extend({
                 basic_composer.mention_set_enabled_commands(commands);
                 basic_composer.mention_set_prefetched_partners(partners);
             });
+            basic_composer.on('keyup_escape', self, function () {
```

Comment:
you should not bind an event handler on the basic composer. The event should come from a trigger_up, so, it should be bound in the custom_events key on top of this file.